ext_localconf.php

```diff
@@ -3,14 +3,14 @@
 	die ('Access denied.');
 }
 
-define ( 'PATH_tx_update_refindex', \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::extPath ( $_EXTKEY ) );
+define ( 'PATH_tx_update_refindex', \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::extPath ( 'update_refindex' ) );
 

```

Comment:
Here I would also keep it as variable, to avoid the risk of mistyping it later.

Reply:
I checked the line, the constant is not used anymore, so I removed the whole line.